app/scripts/views/sign_up.js

```diff
@@ -26,6 +27,7 @@ define(function (require, exports, module) {
   const SignUpMixin = require('views/mixins/signup-mixin');
   const Template = require('stache!templates/sign_up');
 
+  var device = new Device();
```

Comment:
Hi @vladikoff thank you for the review. I have thought of two ways for doing this.
1. Copying the logic from [this](https://github.com/mozilla/fxa-content-server/blob/f44dcb5829ce657d48cb710424df344cfc5bd021/app/scripts/views/connect_another_device.js#L181:L248) file.
2. Extracting the function _hasWebChannelSupport() in other file and including this file in both the places.
Which one would you prefer? And in second case where should I define the function?